internal/lsp/completions/refs/used.go

```diff
@@ -0,0 +1,121 @@
+package refs
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/bundle"
+	"github.com/open-policy-agent/opa/rego"
+
+	rbundle "github.com/styrainc/regal/bundle"
+	rio "github.com/styrainc/regal/internal/io"
+	"github.com/styrainc/regal/pkg/builtins"
+	"github.com/styrainc/regal/pkg/config"
+)
+
+// pq is a prepared query that finds ref names used in a module.
+// pq is prepared at init time to make this functionality more
+// efficient. In local testing, this reduced time by ~95%.
+//
+//nolint:gochecknoglobals
+var pq *rego.PreparedEvalQuery
+
+func init() {
+	regalRules := rio.MustLoadRegalBundleFS(rbundle.Bundle)
+
+	dataBundle := bundle.Bundle{
+		Manifest: bundle.Manifest{
+			Roots:    &[]string{"internal"},
+			Metadata: map[string]any{"name": "internal"},
+		},
+		Data: map[string]any{
+			"internal": map[string]any{
+				"combined_config": map[string]any{
+					"capabilities": rio.ToMap(config.CapabilitiesForThisVersion()),
+				},
+			},
+		},
+	}
+
+	regoArgs := []func(*rego.Rego){
+		rego.ParsedBundle("regal", &regalRules),
+		rego.Module("module.rego", `
+package lsp.completions
```

Comment:
I'd much prefer it if this was a .rego file embedded in the binary. That way we can work with it as any other Rego... lint, test, so on. Was there anything that prevented that?

Reply:
Yeah I should do that. It was only 3 lines or something at the start but it got more involved as I discovered some edge cases... 

Fixed in 88e76666c426da73f737aa4fa00750761d3614bb

Reply:
Nice, thanks!